Content.Server/Connection/ConnectionManager.cs

```diff
@@ -126,6 +133,12 @@ private async Task NetMgrOnConnecting(NetConnectingArgs e)
                 var overallTime = ( await _db.GetPlayTimes(e.UserId)).Find(p => p.Tracker == PlayTimeTrackingShared.TrackerOverall);
                 var haveMinOverallTime = overallTime != null && overallTime.TimeSpent.TotalHours > minOverallHours;
 
+                // Use the custom reason if it exists & they don't have the minimum time
+                if (customReason != string.Empty && !haveMinOverallTime)
+                {
+                    return (ConnectionDenyReason.Panic, customReason, null);
```

Comment:
as before, this will also show if the showReason value is false. Do the same as the above one.